Main/src/init_share.ts

```diff
@@ -145,7 +145,9 @@ export default function initShareButton(
 
     return (newContent: ShareDatas) => {
         content = newContent;
-        if ("canShare" in navigator && navigator.canShare(content.webShare)) {
+        if ("canShare" in navigator &&
+        +            navigator.canShare(content.webShare) &&
+        +            false) {
```

Comment:
Why this code is needed?

Reply:
now I think It is not necessary. 
Its ok to removing this code. thankyou!

Reply:
Dropping commit cf13f450341a476358e7ff2835359a46cf9e6ae0 would fix this, as I think.